### DISCUSSION TITLE
IR Screen Revival

### DISCUSSION THREAD

**Discussion by Sonic-TV** [emoose/OutRun2006Tweaks#269] [Suggestions]
Any chance to revive This screen for IR?
There're still active IRs going on.
![Image_1720284402543](https://github.com/emoose/OutRun2006Tweaks/assets/12370185/06208099-8278-42f7-8244-b134191f2db5)
For IR Info:
http://hinapiyo.sakura.ne.jp/ir/index.cgi

[you probably need to force your browser to load in HTTP rather than HTTPS.]

**Comment by Sonic-TV** [comment]
extra information:
If you check the file "spr_sprani_ending_cvt_e.xst" there're the graphics for the IR screen.
Here is what it looks like (partial extract from 2SP SDX).
![image](https://github.com/user-attachments/assets/60cbf903-f604-4f55-8955-9ca9a693d8cb)

This is also present in C2C's sprite folder.

**Comment by Sonic-TV** [comment]
The font for the password is Seurat by Fontworks with slight modifications (notice the slashed zero.)

**Comment by Sonic-TV** [comment]
![7-texture](https://github.com/user-attachments/assets/887dc1be-2ff1-4470-8a45-bcdc813b48a7)

![13-texture](https://github.com/user-attachments/assets/46238077-ce86-4425-b4bd-f67e1aadc091)
here are the 2 font files that IR uses for displaying information and password, both from lindbergh.
may require extra XST or SZ packs for loading i guess...

**Comment by emoose** [comment]
Is the password system in Lindbergh any different to the one in Chihiro 2/2SP?

Found the password code in lindbergh but wasn't able to find the same code inside C2C sadly, if OR2/2SP used an earlier system it might include that instead though.

Is there any info about how the password gets decoded? Maybe we could just hack in something to display it on screen like how cdswitcher works...

**Comment by Sonic-TV** [comment]
> Is the password system in Lindbergh any different to the one in Chihiro 2/2SP?
> 
> Found the password code in lindbergh but wasn't able to find the same code inside C2C sadly, if OR2/2SP used an earlier system it might include that instead though.
> 
> Is there any info about how the password gets decoded? Maybe we could just hack in something to display it on screen like how cdswitcher works...

Not sure for 2, but no difference for 2SP.
Maybe I'm wrong if you find the chihiro version uses a different one though.
There's no information for password sadly, only some information...

**Comment by Sonic-TV** [comment]
The 16 characters holds these information:
Player name/initials
Player score (not for Time Attack)
Player time
Run type (Outrun, Heart Attack, Time Attack)
Goal and Route (ie which goal point the run ended at and the route that was taken, not applicable to 15-course)
Probably the segment scores as well.
 
not sure what letters are unused, but only uppercase letters and numbers are used, dashes and exclamation marks are used.

**Comment by Sonic-TV** [comment]
Some notice: If you wanna submit your scores to hina's leaderboards (which uses official data from SEGA and still accepting new scores), this screen is **REQUIRED** for sending your scores to them.
Or the Name Entry screen if you're doing time trials in OUTRUN mode.

**Comment by Sonic-TV** [comment]
Oh, right...
The IR Screen has your segment information (per-stage scores or per-sector time) so you can check them on that screen as well...
